Gemfile.lock

```diff
@@ -103,7 +103,7 @@ GEM
     minitest (5.15.0)
     msgpack (1.4.5)
     nio4r (2.5.8)
-    nokogiri (1.13.1-x86_64-darwin)
+    nokogiri (1.13.1-x86_64-linux)
```

Comment:
I have a feeling that we're going to be flip-flopping this dependency based on everyones environment. I think we should either not add this change, or explore a common dev env using something like docker to reduce this kind of noise.

Reply:
Was it me who changed version? How could it happened?

Reply:
Changed to nokogiri (1.13.1-x86_64-darwin).  Any questions please ask.